src/main/java/Duke.java

```diff
@@ -1,10 +1,224 @@
+import duke.*;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
+/**
+ * Duke Program behaves like a chatbot that interacts with user input.
+ * It allows the user to create a list of Tasks consisting of todos, deadlines, and events
+ * @author oliverloo
+ * @version 1.0
+ *
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    /**
+     * Main method to initiliase duke chatbot
+     * @param args
+     */
+        public static void main (String[]args) {
+            Tasklist list = new Tasklist();
+            try {
+                Storage.loadFile(list);
+            } catch(Exception e) {
+                System.out.println(e.getMessage());
+            }
+
+                Ui.greet();
+                Scanner sc = new Scanner(System.in);
+                //echo
+
+                while (true) {
+                    try {
+                    String input = sc.nextLine();
+
+                    if (Parser.is_Bye(input)) {
+                        Ui.displayByeMessage();
+                        break;
+                    }
+
+                    //list
+                    else if (Parser.is_List(input)) {
+                        Duke.executeList(list);
+                    }
+
+                    //mark
+                    else if (Parser.is_Mark(input)) {
+                        Duke.executeMark(input, list);
+                    }
+
+                    //unmark
+                    else if (Parser.is_Unmark(input)) {
+                        Duke.executeUnmark(input, list);
+                    }
+
+                    //delete
+                    else if (Parser.is_Delete(input)) {
+                        Duke.executeDelete(input, list);
+                    }
+
+                    //clear
+                    else if (Parser.is_Clear(input)) {
+                        Duke.executeClear(input, list);
+                    }
+
+                    //todo
+                    else if (Parser.is_toDo(input)) {
+                        Duke.executeToDo(input, list);
+                    }
+
+                    //deadline
+                    else if (Parser.is_Deadline(input)) {
+                        Duke.executeDeadline(input, list);
+                    }
+
+                    //event
+                    else if (Parser.is_Event(input)) {
+                        Duke.executeEvent(input, list);
+                    }
+
+                    //find
+                    else if (Parser.is_Find(input)) {
+                        Duke.executeFind(input, list);
+                    }
```

Comment:
It's good that the boolean method names starts with an is, but I think the method names could be corrected to use camelCase to adhere to the coding style, for example isList(), isBye() etc.